content/en/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md

```diff
@@ -24,9 +26,10 @@ vacated by the evicted critical add-on pod or the amount of resources available
 
 ### Marking pod as critical
 
-To be considered critical, the pod has to run in the `kube-system` namespace (configurable via flag) and
+Prior to v1.11, critical pod has to run in the `kube-system` namespace, this restriction was removed after v1.11 and pod in any namespace can be configed as a critical pod by the following either way:
 
-* Have the priorityClassName set as "system-cluster-critical" or "system-node-critical", the latter being the highest for entire cluster. Alternatively, you could add an annotation `scheduler.alpha.kubernetes.io/critical-pod` as key and empty string as value to your pod, but this annotation is deprecated as of version 1.13 and will be removed in 1.14.
+* Ensure the PodPriority [feature gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) is enabled, have the priorityClassName set as "system-cluster-critical" or "system-node-critical", the latter being the highest for entire cluster. Available in v1.10+
```

Comment:
Critical pod priorityClass addition #58835
https://github.com/kubernetes/kubernetes/pull/58835

https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.10.md#scheduling
All pods with priorityClassName system-node-critical and system-cluster-critical will be critical pods while preserving backwards compatibility. (#58835, @ravisantoshgudimetla)

Reply:
What is available since v1.10? PodPriority? The priority class names? Has the annotation been removed already (sometimes things got deviated from the schedule)?

Reply:
The priority class names available since v1.10(I'll reword this line to make it clear). 
The annotation [have not been removed actually](https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L58) and still [being in use](https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/priority/admission.go#L187-L190).